rclrs/src/node/mod.rs

```diff
@@ -88,6 +89,87 @@ impl Node {
         })
     }
 
+    /// Returns the name of the node.
+    ///
+    /// This returns the name after remapping, so it is not necessarily the same as the name that
+    /// was used when creating the node.
+    ///
+    /// # Example
+    /// ```
+    /// # use rclrs::{Context, RclReturnCode};
+    /// // Without remapping
+    /// let context = Context::new([])?;
+    /// let node = context.create_node("my_node")?;
+    /// assert_eq!(node.name(), String::from("my_node"));
+    /// // With remapping
+    /// let remapping = ["--ros-args", "-r", "__node:=your_node"].map(String::from);
+    /// let context_r = Context::new(remapping)?;
+    /// let node_r = context_r.create_node("my_node")?;
+    /// assert_eq!(node_r.name(), String::from("your_node"));
+    /// # Ok::<(), RclReturnCode>(())
+    /// ```
+    pub fn name(&self) -> String {
+        self.get_string(rcl_node_get_name)
+    }
+
+    /// Returns the namespace of the node.
+    ///
+    /// This returns the namespace after remapping, so it is not necessarily the same as the
+    /// namespace that was used when creating the node.
+    ///
+    /// # Example
+    /// ```
+    /// # use rclrs::{Context, RclReturnCode};
+    /// // Without remapping
+    /// let context = Context::new([])?;
+    /// let node = context.create_node_with_namespace("/my/namespace", "my_node")?;
+    /// assert_eq!(node.namespace(), String::from("/my/namespace"));
+    /// // With remapping
+    /// let remapping = ["--ros-args", "-r", "__ns:=/your_namespace"].map(String::from);
+    /// let context_r = Context::new(remapping)?;
+    /// let node_r = context_r.create_node("my_node")?;
+    /// assert_eq!(node_r.namespace(), String::from("/your_namespace"));
+    /// # Ok::<(), RclReturnCode>(())
+    /// ```
+    pub fn namespace(&self) -> String {
+        self.get_string(rcl_node_get_namespace)
+    }
+
+    /// Returns the fully qualified name of the node.
+    ///
+    /// The fully qualified name of the node is the node namespace combined with the node name.
+    /// It is subject to the remappings shown in [`Node::name`] and [`Node::namespace`].
+    ///
+    /// # Example
+    /// ```
+    /// # use rclrs::{Context, RclReturnCode};
+    /// let context = Context::new([])?;
+    /// let node = context.create_node_with_namespace("/my/namespace", "my_node")?;
+    /// assert_eq!(node.fully_qualified_name(), String::from("/my/namespace/my_node"));
+    /// # Ok::<(), RclReturnCode>(())
+    /// ```
+    pub fn fully_qualified_name(&self) -> String {
+        self.get_string(rcl_node_get_fully_qualified_name)
+    }
+
+    fn get_string(
```

Comment:
This looks like a good candidate for a macro, IMHO

Reply:
I don't care much either way in this instance, but the approach using a function could be said to have a few advantages:
- Less code size (obviously doesn't matter much for such a small function)
- No need to know `macro_rules!`
- Better [IDE support](https://rust-analyzer.github.io/blog/2021/11/21/ides-and-macros.html)
- You can see the type of the function supported by the helper
- Definition of the helper is closer to the call site (since macro would typically be at the top level)

Reply:
It's a little less efficient though, but I think for these functions that won't matter.

Reply:
Yeah, that makes sense. I was just thinking of https://github.com/ros2-rust/ros2_rust/issues/141 and how to integrate it with this, though.

Reply:
Yes, that's something we'll need to think about. Maybe `call_ffi!(getter, …)` will already work if the right token type is used.